README.md

```diff
@@ -94,6 +94,35 @@ The return value is an instance of a `videojs` [`Player`](https://docs.videojs.c
 ### Setting a source
 If the player you bundled is configured in studio, or the video element you intialized had a source then the player should be ready to go. If your player does not have a source you can add one using `player.src({src: 'some-url.mp4', type: 'video/mp4'})`.
 
+### Putting it all (Example)
+1. Setup the plugin with webpack for the player that you want.
+2. Make sure there are no video elements that will auto-setup on the html page you are going to test with, unless you want that. See the auto-setup section for more info on that.
+3. Setup the player in your webpack entry point (or anywhere really)
+
+```js
+import window from 'global/window';
+import document from 'global/document';
```

Comment:
I think we should probably remove these two lines. Could be confusing to people who don't use the `global` package or those aliases.

Reply:
I will remove the lines and mention the global package in a comment, just in case